docs/sebak_command.md

```diff
@@ -0,0 +1,168 @@
+To install `sebak` command, please check the [installation guide](./sebak_Installation.md).
```

Comment:
File name link(sebak_Installation.md) should be also modified.

Reply:
I changed it.